### PR TITLE
Bug fix + documentation mismatch fix

### PR DESCRIPTION
**Pull request by oliver-walker-hms**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -239,4 +246,4 @@ The "PostDuplicateTagValues" has been added to the configuration.  This option c
 + Batch based datapoint posting
 
 ## Version 0.1.0
-Initial Release
+Initial Release
```
Comment (TomKimsey) [inline CHANGELOG.md:249]: unrelated ws change

File: pom.xml [hunk 2]
```diff
@@ -427,4 +427,4 @@
       <version>1.14.2</version>
     </dependency>
   </dependencies>
-</project>
+</project>
```
Comment (TomKimsey) [inline pom.xml:430]: unrelated ws change

File: starting-files/jvmrun [hunk 3]
```diff
@@ -1 +1 @@
--heapsize 25M -classpath /usr/flexy-osisoft-connector-3.0.14-full.jar -emain com.hms.flexyosisoftconnector.OSIsoftConnectorMain
+-heapsize 25M -classpath /usr/flexy-osisoft-connector-3.0.15-full.jar -emain com.hms.flexyosisoftconnector.OSIsoftConnectorMain
```
Comment (TomKimsey) [inline starting-files/jvmrun:1]: unrelated ws change

File: src/main/java/com/hms/flexyosisoftconnector/dataserver/OSIsoftServer.java [hunk 4]
```diff
@@ -30,18 +30,6 @@ public class OSIsoftServer {
...
-
-  /** Ewon error code representing a JSON error */
-  public static final int JSON_ERROR = -100;
-
-  /** Ewon error code representing a web ID error */
-  public static final int WEB_ID_ERROR = SCHttpUtility.HTTPX_CODE_AUTH_ERROR;
-
-  /** Ewon error code representing a generic error */
-  public static final int GENERIC_ERROR = 254;
-
```
Comment (TomKimsey) [inline src/main/java/com/hms/flexyosisoftconnector/dataserver/OSIsoftServer.java:44]: What is the value in removing these? could any of these be possibility needed in the future? In general im fine with leaving in unused constants that are ewon specific like this. Really I think its probably worth having an Ewon error constants file in the extension lib but thats a different can of worms.

File: src/main/java/com/hms/flexyosisoftconnector/dataserver/OSIsoftServer.java [hunk 5]
```diff
@@ -551,7 +547,7 @@ public static boolean postOcsBatch(String payload) {
   public static boolean postBatch(String payload) {
     boolean requestSuccess =
         RequestHttps(
-            OSIsoftConfig.getTargetURL() + "batch/",
+            OSIsoftConfig.getServerUrl() + "/batch",
```
Comment (TomKimsey) [inline src/main/java/com/hms/flexyosisoftconnector/dataserver/OSIsoftServer.java:550]: ending "/" removed

**Review by TomKimsey — CHANGES_REQUESTED** [review]

File: src/main/java/com/hms/flexyosisoftconnector/dataserver/OSIsoftServer.java (hunk 5 above)
Comment (oliver-walker-hms) [reply src/main/java/com/hms/flexyosisoftconnector/dataserver/OSIsoftServer.java:550]: This was intentional. The API works with or without the trailing slash, but all other requests in the connector do not end with a trailing slash. I can move this to another commit to include a commit message about this

File: src/main/java/com/hms/flexyosisoftconnector/dataserver/OSIsoftServer.java (hunk 4 above)
Comment (oliver-walker-hms) [reply src/main/java/com/hms/flexyosisoftconnector/dataserver/OSIsoftServer.java:44]: This was a code cleanup focused change, focused on decreasing errant code. I'm fine leaving these in if there is an argument for it, will drop this commit.

**Review by alexjhawk — APPROVED** [review]

**Review by TomKimsey — APPROVED** [review]
